python/lsst/daf/butler/registry/databases/sqlite.py

```diff
@@ -65,15 +65,16 @@ class _Replace(sqlalchemy.sql.Insert):
     pass
 
 
+# SQLite and PostgreSQL use similar syntax for their ON CONFLICT extension,
+# but SQLAlchemy only knows about PostgreSQL's, so we have to compile some
```

Comment:
That seems odd that sqlite doesn't support it in sqlalchemy. Is it just that sqlalchemy team haven't got around to it or there are problems with it? Could we submit a patch?

Reply:
I think it's _slightly_ new functionality in SQLite, but not that new.  I imagine a patch would be appreciated, but it'd be a while before I got around to it.